integration_tests/ethereum/contracts/CosmosToken.sol

```diff
@@ -46,14 +47,15 @@ contract CosmosERC20 is ERC20 {
 
 
 	/**
-	 * @dev Sets the gravity contract to a new address.
+	 * @dev Sets the gravity contract and transfer the balance to the new address.
 	 *
 	 * Requirements:
 	 *
 	 * - `msg.sender` must be the current gravity contract
 	 */
 	function setGravityContract(address _gravityAddress) external onlyGravity {
-
+		require(balanceOf(_gravityAddress) == 0, "new gravity address balance should be zero");
+		_transfer(gravity, _gravityAddress, balanceOf(gravity));
```

Comment:
why it needs to transfer?

Reply:
this is related to this issue

https://github.com/crypto-org-chain/gravity-bridge/pull/93

this method only set the admin but the managed balance are not transferred